eosdis-nasa/earthdata-pub-dashboard
# removing refrences to mEditor

# Description

Removes references to mEditor

## Spec

See Ticket: [EDPUB-1245](https://bugs.earthdata.nasa.gov/browse/EDPUB-1245)

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Confirm workflow work as expected
---

## Change Log

_(copy/paste-able change log notes. check the box when the change is also in CHANGELOG.md)_

* [Add _____](link to commitid)
* [Fix _____](link to commitid)
